index.html

```diff
@@ -19,42 +19,205 @@
+                <li class="html technology">html</li>
+              </ul>
+              <button class="project-btn">See Project</button>
+            </div>
+          </div>
+          <div class="single_project_card card-4">
+            <div class="project_img_placeholder"></div>
+            <div class="project_title_container">
+              <h3 class="project_title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="technologies_container">
+                <li class="ruby technology">Ruby on rails</li>
+                <li class="css technology">css</li>
+                <li class="javascript technology">JavaScript</li>
+                <li class="html technology">html</li>
+              </ul>
+              <button class="project-btn">See Project</button>
+            </div>
+          </div>
+          <div class="single_project_card card-5">
+            <div class="project_img_placeholder"></div>
+            <div class="project_title_container">
+              <h3 class="project_title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="technologies_container">
+                <li class="ruby technology">Ruby on rails</li>
+                <li class="css technology">css</li>
+                <li class="javascript technology">JavaScript</li>
+                <li class="html technology">html</li>
+              </ul>
+              <button class="project-btn">See Project</button>
+            </div>
+          </div>
+          <div class="single_project_card card-6">
+            <div class="project_img_placeholder"></div>
+            <div class="project_title_container">
+              <h3 class="project_title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="technologies_container">
+                <li class="ruby technology">Ruby on rails</li>
+                <li class="css technology">css</li>
+                <li class="javascript technology">JavaScript</li>
+                <li class="html technology">html</li>
+              </ul>
+              <button class="project-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- About myself section -->
+      <section id="about_myself_container">
+        <div class="about_me">
+          <div class="top-illustration"></div>
+          <div class="bottom-illustration"></div>
+          <!-- <img src="./images/about-me-illustration-top.png" alt="" />
+          <img src="./images/about-me-illustration-bottom.png" alt="" /> -->
+          <h1 class="about_me_heading">About me</h1>
+          <p class="about_me_description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button class="resume_btn">Get My Resume</button>
+        </div>
```

Comment:
- [ Required ]  Please consider adding the background images that are present in the `About me` section in the Figma design. You can achieve this by using the `background-image` and `background-position` css properties. Please see the screenshot below for more details on this.
## Figma design 
![Screenshot from 2022-12-02 14-11-09](https://user-images.githubusercontent.com/16392046/205280096-e3800aed-40e2-4f4e-b3f1-902dd37a01d0.png)

## Your Implementation
<img src="https://user-images.githubusercontent.com/16392046/205279586-2e21cf36-b2d4-48a1-afe2-9c912a90c148.png" width="248px" />